### ISSUE TITLE
problems when i build the project

### ISSUE THREAD

**Issue by Ayman-SHaaban-ELsayed** [JustWayward/BookReader#253]
i face this problems when i build the project please help:
1-
`Caused by: groovy.lang.MissingPropertyException: Could not set unknown property 'includeCompileClasspath' for AnnotationProcessorOptions$AgpDecorated_Decorated{classNames=[], arguments={}, compilerArgumentProviders=[]} of type com.android.build.gradle.internal.dsl.AnnotationProcessorOptions$AgpDecorated`
this problem refers to this line in gradle file:
```
 javaCompileOptions {
            annotationProcessorOptions {
                includeCompileClasspath = true           //<<here the problem
            }
        }
    }
```
2-build gradle for EasyAdapterLibrary in the app needs to update.
3-Android Manifest gives many error ,please try to run the project and solve this problems.
4-   this line in build.gradle(BookreaderMaster 
`     maven { url 'http://maven.aliyun.com/nexus/content/groups/public/' }//阿里云镜像服务
`
gives this error:
` Using insecure protocols with repositories, without explicit opt-in, is unsupported. Switch Maven repository 'maven(http://maven.aliyun.com/nexus/content/groups/public/)' to redirect to a secure protocol (like HTTPS) or allow insecure protocols. See https://docs.gradle.org/7.5.1/dsl/org.gradle.api.artifacts.repositories.UrlArtifactRepository.html#org.gradle.api.artifacts.repositories.UrlArtifactRepository:allowInsecureProtocol for more details. `
5-please update the     **target Sdk Version**  to a higher version like: 32 to be suitable for the new devices .

**Comment by smuyyh** [comment]
I updated the gradle and mainfest 2 days ago , and it can run normally. 

```
Android Studio Bumblebee | 2021.1.1 Patch 2
Build #AI-211.7628.21.2111.8193401, built on February 17, 2022
Runtime version: 11.0.11+0-b60-7590822 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.3.1
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 12
Registry: external.system.auto.import.disabled=true, ide.balloon.shadow.size=0, ide.images.show.chessboard=true
Non-Bundled Plugins: manjaro.mpb (1.7), com.wudaokou.hippo.HMPlugin (1.8.3), org.intellij.plugins.markdown (211.7142.37)
```

however, the api request used in the project was acquired through packet capturing long ago and has been deprecated, which will get blank view.

so, this project can no longer be maintained

**Comment by Ayman-SHaaban-ELsayed** [comment]
Can  you please update the manifest please.

The project is large, if i need only to read the local files ,which directions i can use from your project?🌷

**Comment by smuyyh** [comment]
read the local files ? do you mean reading local book txt files?

**Comment by Ayman-SHaaban-ELsayed** [comment]
Yes  i need only to read the local files like txt and pdf,which classes and directions  i can use from your project?🌷
And do not forget to update the manifest please.

**Comment by smuyyh** [comment]
I have not encountered any errors in the mainfest file, you can check the specific error and try to fix it.

**Comment by smuyyh** [comment]
read txt ：https://github.com/smuyyh/BookReader/blob/master/app/src/main/java/com/justwayward/reader/ui/activity/ReadActivity.java#L417

read pdf: https://github.com/smuyyh/BookReader/blob/master/app/src/main/java/com/justwayward/reader/ui/activity/ReadPDFActivity.java

**Comment by Ayman-SHaaban-ELsayed** [comment]
> I have not encountered any errors in the mainfest file, you can check the specific error and try to fix it.

when i set     compileSdk to 32 the problems happend,please update the targetSdk and compileSdk  and run the project.
Thank you for your patience🌹🌹

**Comment by Ayman-SHaaban-ELsayed** [comment]
> > I have not encountered any errors in the mainfest file, you can check the specific error and try to fix it.
> 
> when i set compileSdk to 32 the problems happend,please update the targetSdk and compileSdk and run the project. Thank you for your patience🌹🌹

can you help please🌹❤️

**Comment by Ayman-SHaaban-ELsayed** [comment]
> > I have not encountered any errors in the mainfest file, you can check the specific error and try to fix it.
> 
> when i set compileSdk to 32 the problems happend,please update the targetSdk and compileSdk and run the project. Thank you for your patience🌹🌹

can you help please🌹❤️
